.gitignore

```diff
@@ -40,3 +40,4 @@ custom_plan.rb
 pkg/
 config/hooks
 *.csv
+.envrc
```

Comment:
I mean is there a trick you use in the file to set some variables? Share it please.